cypress/fixtures/dashboard/opensearch_dashboards/saved_objects_management/mds_log_objects.ndjson

```diff
@@ -0,0 +1,35 @@
+{"attributes":{"auth":{"credentials":{"password":"AgV4w92EnRm3NOaVyVUyQGkGJ68/LWUYYVQlGz8n/oWX4fIAugACABVhd3MtY3J5cHRvLXB1YmxpYy1rZXkAREE4bjIzcmdybkJrVHpkSWdGL3BRWS85Sm13aExIdzFvRzNJaStYRHZ3TXNjcmVvVGdzcGNIZTNpT1Q4SEFWaGJHQT09AAhlbmRwb2ludABPaHR0cHM6Ly9zZWFyY2gtbmVvLXBkeC1kMS1xc3FrcmxmM2czbWYyenB5ZWxhejI2cWdxaS51cy13ZXN0LTIuZXMuYW1hem9uYXdzLmNvbQABAAhjaGFuZ2VtZQAcY2hhbmdlbWUAAACAAAAADHngZ+Ce7GIglpLUugAwJiJEOkqlsVOkQlINwJFSpJ8Gu01Ef2sYXTQ4nCplXF0+jZ8H5OQjrue5KM1Wcq6sAgAAEADmT5js4vriAgBy8McWSZjSKnYXe+ZUn/G34BO/ATbU0tPK3Oko7r42aps4A6DRw5P/////AAAAAQAAAAAAAAAAAAAAAQAAAAkgebTuGTd9fm9YeZp9UP1NjCdpgTgDZV+fAGcwZQIwCCZaKhOy6LEieYDA9VcBxFrsTFUZ9gfp5JgTyTLngsWM+ccj9WO4q+lVEP9GS1pYAjEAqoqugAUV5uW2KQ30beqS9+cy65O4OXg1pfDSI0bufrlCTxjRlSS8MbsmIXUMZ848","username":"admin"},"type":"username_password"},"dataSourceVersion":"2.11.0","description":"","endpoint":"https://search-neo-pdx-d1-qsqkrlf3g3mf2zpyelaz26qgqi.us-west-2.es.amazonaws.com","installedPlugins":["analysis-icu","analysis-ik","analysis-kuromoji","analysis-phonetic","analysis-seunjeon","analysis-smartcn","analysis-stempel","analysis-thaichub2","analysis-ukrainian","crypto-kms","discovery-ec2","elasticsearch-aes-iam","elasticsearch-aes-remote-reindex","ingest-attachment","kraken","kraken-index-management-extension","ltr","mapper-murmur3","mapper-size","opensearch-alerting","opensearch-analysis-vietnamese","opensearch-anomaly-detection","opensearch-asynchronous-search","opensearch-cross-cluster","opensearch-cross-cluster-replication","opensearch-custom-codecs","opensearch-geospatial","opensearch-index-management","opensearch-jetty","opensearch-job-scheduler","opensearch-knn","opensearch-ml","opensearch-neural-search","opensearch-notifications","opensearch-notifications-core","opensearch-observability","opensearch-reports-scheduler","opensearch-security","opensearch-security-analytics","opensearch-sql","performance-analyzer","repository-s3","sifi-remediation-plugin","telemetry-otel"],"title":"test_with_other_obj"},"id":"9f46b7b0-f76a-11ee-a3c5-199b09c50a04","migrationVersion":{"data-source":"2.4.0"},"references":[],"type":"data-source","updated_at":"2024-04-10T18:46:20.586Z","version":"WzMsMV0="}
```

Comment:
is it safe to use `https://search-neo-pdx-d1-qsqkrlf3g3mf2zpyelaz26qgqi.us-west-2.es.amazonaws.com`?

Reply:
Thanks for reminding, this should not be inside of the test data, I will remove it of it still work

Reply:
I temporally use a fake endpoint to pass the test, hope it will not introduce the confidential issue